cardano-api/src/Cardano/Api/Eras.hs

```diff
@@ -137,6 +138,12 @@ deriving instance Eq   (CardanoEra era)
 deriving instance Ord  (CardanoEra era)
 deriving instance Show (CardanoEra era)
 
+instance ToJSON (CardanoEra era) where
+   toJSON ByronEra   = "Byron"
```

Comment:
:+1:

Reply:
This would be the perfect output also for the query tip or query protocol-parameters about the current era the node is in right now.

Reply:
@gitmachtl Could you please open an issue for that?

Reply:
> @gitmachtl Could you please open an issue for that?

https://github.com/input-output-hk/cardano-node/issues/2509

already there as a FR